book/getting-started/run-a-wallaroo-application.md

```diff
@@ -48,16 +48,16 @@ If you need to start the UI after stopping it, run:
 docker start mui
 ```
 
-## Shell 2: Run Giles Receiver
+## Shell 2: Run Data Receiver
 
-We'll use Giles Receiver to listen for data from our Wallaroo application.
+We'll use Data Receiver to listen for data from our Wallaroo application.
 
 ```bash
-cd ~/wallaroo-tutorial/wallaroo/giles/receiver
-./receiver --listen 127.0.0.1:5555 --no-write --ponythreads=1 --ponynoblock
+cd ~/wallaroo-tutorial/wallaroo/utils/data_receiver
+./data_receiver --listen 127.0.0.1:5555 --no-write --ponythreads=1 --ponynoblock
 ```
 
-You should see the `Listening for data` that indicates that Giles receiver is running.
+Data Receiver will start up and receive data without creating any output. By default, it prints received data to standard out, but we are giving it the `--no-write` flag which results in no output.
```

Comment:
Don't we want users to see the output in the example?

Reply:
I followed the example from the Go documentation: https://github.com/WallarooLabs/wallaroo/blob/master/book/go/getting-started/run-a-wallaroo-go-application.md#shell-2-run-the-data-receiver